src/components/Book.js

```diff
@@ -10,14 +10,22 @@ const Book = () => {
   }, [dispatch]);
 
   const books = useSelector((state) => state.books);
+  console.log(books);
   return (
-    <div>
-      {books.map(({ title, author, itemId }) => (
-        <li key={itemId}>
+    <div className="books">
+      {books.map(({
+        title, author, itemId, category,
+      }) => (
+        <li key={itemId} className="book__li">
+          <h5>{category}</h5>
           <h3>{title}</h3>
           <p>{author}</p>
           <button type="submit" onClick={() => dispatch(deleteBook(itemId))}>
-            remove
+            |
+            {' '}
+            <span>Remove</span>
+            {' '}
+            |
           </button>
         </li>
       ))}
```

Comment:
- [ ] Kindly note that you are to replicate the design exactly. Ensure you add the other remaining styling as contained in the Zeplin design.

  Your Implementation | Design
  :---:|:---:
  ![ejur](https://user-images.githubusercontent.com/54848363/201293860-61e0b205-057f-4740-bf4d-eb7aa3bbeb46.jpg) | ![fjfw](https://user-images.githubusercontent.com/54848363/201293866-0daac6bb-3730-4aee-ab86-69103b4143a9.jpg)